### ISSUE TITLE
More documentation 

### ISSUE THREAD

**Issue by Iamtheoldman** [satelerd/GPT3-Models#1]
not working need more info

**Comment by satelerd** [comment]
You need your own api key to make a request and run gpt3.
It can be found here: https://beta.openai.com/account/api-keys

**Comment by Iamtheoldman** [comment]
Yeah we have the key but your file doesn’t specify where to put the key or
what is the key name
